hawkbit-rest-core/src/main/java/org/eclipse/hawkbit/rest/SecurityManagedConfiguration.java

```diff
@@ -39,7 +37,6 @@
 @Slf4j
 @Configuration
 @EnableWebSecurity
-@EnableGlobalMethodSecurity(prePostEnabled = true, mode = AdviceMode.ASPECTJ, proxyTargetClass = true, securedEnabled = true)
```

Comment:
isnt here a change in behavior ? mode = AdviceMode.ASPECTJ wheres by default mode is PROXY ?

Reply:
@PreAuthenticate annotations are used only in repository and management API
that's why the EnableMethod security is moved only in repository and management API configurations

Reply:
and regarding removal of ASPECTJ - it seems this mode doesn't work - and it was the server app annotations with  PROXY mode that really makes things to work.